server/app.js

```diff
@@ -6,15 +6,14 @@ const logger = require("morgan");
 const mongoose = require("mongoose");
 require('dotenv').config({ path: './.env'})
 
-const indexRouter = require("./routes/index");
-const pingRouter = require("./routes/ping");
 const signupRouter = require("./routes/signup");
 const signinRouter = require("./routes/signin");
+const JWTRouter = require("./routes/JWT")
 const passport = require('passport')
 
 const { json, urlencoded } = express;
 
-const app = express();
+var app = express();
```

Comment:
```suggestion
const app = express();
```
var should not be used in modern ES6, FYI